website/content/api-docs/secret/pki/issuance.mdx

```diff
@@ -750,6 +751,10 @@ updated values as a response along with an updated `last_updated` field.
  `locality`, `province`, `street_address`, `postal_code`, `serial_number`, `use_pss`, `key_type`, `key_bits`,
  `add_basic_constraints`
 
+- `disabled_validations` `(list: ["DisableMatchingKeyIdValidation", "DisableCertTimeValidation"])` - Checks to
```

Comment:
normally we put the default in the quotes.  These validations aren't disabled by default.

Reply:
Ah, missed that, I will push a fix.